shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -135,6 +135,12 @@ void PlatformViewChannel::HandleMethodCall(
         focused_view->second->SetFocus(false);
       }
 
+      auto existing_view = view_instances_.find(view_id);
+      if (existing_view != view_instances_.end()) {
+        existing_view->second->Dispose();
+        view_instances_.erase(existing_view);
+      }
+
```

Comment:
Here doesn't seem a right place to dispose view instances. Only a single view instance can exist per app in this way.